README.md

```diff
@@ -22,9 +42,9 @@ If the issue is a protocol weakness that cannot be immediately exploited or some
 
 ## Install
 
-The canonical download instructions for IPFS are over at: http://ipfs.io/docs/install/
+The canonical download instructions for IPFS are over at: http://ipfs.io/docs/install/. It is **highly suggested** you follow those instructions if you are not interested in working on IPFS development. Otherwise, follow the instructions below.
```

Comment:
I think that we shouldn't suggest one over the other. I don't know the cost benefits of building from source compared to the other option here; I just think that it's not clear enough that people who just want to install IPFS should go to the main website.